### DISCUSSION TITLE
IPv4 not working, even though config seems fine

### DISCUSSION THREAD

**Discussion by HKSubstance** [fakemanhk/openwrt-jp-ipoe#28] [Q&A]
Hello, I've recently moved to a new apartment which allows me to use Hikari, and I'd like to use my Netgear R7800 router with Openwrt to cope with IPoE.

BTW, the provider I signed up is GMOとくとくＢＢ.

Basically, IPv6 seems to be working fine, but IPv4 is having major trouble.
According to the instructions and my understanding, the MAP-E interface should be configured correctly.

In the interface overview I can see that the MAP-E interface has been assigned the correct IPv4 address (same that is assigned to the rental router), and the system log is not showing any apparent errors related to the configuration.

However, whenever I try to open a webpage that doesn't support IPv6, the connection just times out. So basically like 90% of the pages/services I use are not working ....

When trying to ping these websites I can see that the correct IP addresses are resolved. Nothing else seems to be happening though.
The MAP-E interface shows some that some packets are transmitted whenever I try to open a IPv4 website, ping or whatnot, but zero packets are received.

Has anyone experienced these issues and knows how to go about troubleshooting?

As a final note, the behavior is identical on my Windows 10, Linux and iOS clients.

Regards,
![Untitled](https://github.com/fakemanhk/openwrt-jp-ipoe/assets/168643662/6b5a068a-a5d3-4f95-9c5a-d380eb682a3c)

**Comment by LIII-XXII** [comment]
I suspect that the default route for ipv4 is through interface `wan`, where it should be `WAN6MAPE`. You don't need `wan` if you don't use an IP (hikari) phone. In fact, you can even delete it if you don't use the phone.
If you do need it, in the settings for `wan`, make sure "Use default gateway" is unchecked like so:

![image](https://github.com/fakemanhk/openwrt-jp-ipoe/assets/583664/871729a4-0d6b-4f18-bf18-1f383e336228)

**Comment by fakemanhk** [comment]
If you ping the IPv4 address directly from OpenWrt diagnosis page, what can you observe?

**Comment by twatwak** [comment]
same problem. ISP is Biglobe
cannot browse ipv4 websites.
OpenWrt diagnosis page results this.
```
PING openwrt.org (64.226.122.113): 56 data bytes
--- openwrt.org ping statistics ---
5 packets trasmitted, 0 packets received, 100% packet loss
```

**Comment by fakemanhk** [comment]
When does this happen? It worked in the past and then failed just now? Do
you see any suspicious log entries? Need more information here.

**Reply by twatwak** [reply]
i m trying to setup new on proxmox vm following this guide.
no i dont see any suspicious log beside these two.
`ip6_tunnel: map-MAPE xmit: Local address not yet configured!`
`IPv6: eth0: IPv6 duplicate address [ipv6 address] used by [mac address] detected!`

**Comment by fakemanhk** [comment]
"Local address not configured" warning can be ignored.

But you need to solve the duplicate IPv6 address problem first, and if
possible try to use dedicated machine installation to rule out the issue
with Proxmox first.

**Comment by twatwak** [comment]
Another TpLink Router connected in the router mode was the problem.
I removed it and connected it to openwrt lan port in access point mode.
I guess everything is working fine now.
But I dont get IPv6 DNS servers as before from the openwrt router.
Before I used to get NTT ipv6 dns servers
```
IPv6 DNS Servers: aaaa:bbb:cccc:b::3 (Unencrypted)
                    aaaa:bbb:cccc:a::3 (Unencrypted)
```
Now i get openwrt routers ipv6 dns servers
`IPv6 DNS Servers: aaaa:bbbb:cccc::1 (Unencrypted)`
Any Idea?

**Reply by fakemanhk** [reply]
> Now i get openwrt routers ipv6 dns servers
> `IPv6 DNS Servers: aaaa:bbbb:cccc::1 (Unencrypted)`
> Any Idea?

Let's open up another thread/issue (with more config info) to discuss so that we are not mixing different issues in one thread.
